### PR TITLE
Make top-left logo clickable

### PR DESCRIPTION
**Pull request by stewemetal**
I assumed that the <a> tag with the Rainbow Cake text has custom styling, so the logo got wrapped in its own <a> tag.

### CODE REVIEW

**Comment by zsmb13** [comment]
It's actually okay to wrap both in the same <a> tag here, and it makes for a better clickable area (includes the blank space on the sides of the image as well).

File: layouts/partials/logo.html [hunk 1]
```diff
@@ -1,2 +1,4 @@
-<img style="max-height:3em;display: inline-block" src="/images/icon_tight_crop.png" alt="RainbowCake logo"/>
-<a href="/">Rainbow​Cake</a>
+<a href="/">
+  <img style="max-height:3em;display: inline-block" src="/images/icon_tight_crop.png" alt="RainbowCake logo"/>
+</a>
+</a href="/">Rainbow​Cake</a>
```
Comment (zsmb13) [inline layouts/partials/logo.html]: This opening `<a>` tag was broken on this line, though it will likely be removed anyway.

Comment (zsmb13) [inline layouts/partials/logo.html:4]: This opening `<a>` tag was broken on this line, though it will likely be removed anyway.

**Review by zsmb13 — CHANGES_REQUESTED** [review]
It's actually okay to wrap both in the same tag here, and it makes for a better clickable area (includes the blank space on the sides of the image as well).

File: layouts/partials/logo.html (hunk 1 above)
Comment (stewemetal) [reply layouts/partials/logo.html]: My bad. :/

**Comment by stewemetal** [comment]
Thanks for the heads-up!

**Review by zsmb13 — APPROVED** [review]
Looks good now, cheers!

**Pull request merged**